app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -80,11 +79,8 @@ def parse_series(s)
           :cpus           => s['numberOfCores'], # where are the virtual CPUs??
           :cpu_cores      => s['numberOfCores'],
           :memory         => s['memoryInMB'].to_f,
-
-          # Extra keys
-          :os_disk_size   => s['osDiskSizeInMB'] * 1024,
+          :root_disk_size => s['osDiskSizeInMB'] * 1024,
           :swap_disk_size => s['resourceDiskSizeInMB']
```

Comment:
And this is exactly why we test for specific values and not just `.not_to be_nil`.  The latter doesn't really test much at all.